### PR TITLE
Add a POST form to the landing page

### PR DESCRIPTION
**Pull request by SuperQ**
Add configuration to the landing page generator to allow a POST form.
* Reduce header font size.

### CODE REVIEW

**Comment by SuperQ** [comment]
Screenshot example for the snmp_exporter:

image

**Comment by juliusv** [comment]
:+1: Good enough for this purpose :) If you wanted to avoid the input length business, you could also just put the label and input on separate lines (like it's done in most forms nowadays), but this way is also fine.

**Comment by SuperQ** [comment]
Yea, I was just trying to keep the inline format.

**Review by roidelapluie — CHANGES_REQUESTED** [review]

**Comment by roidelapluie** [comment]
My comment was still pending..

File: web/landing_page.html [hunk 1]
```diff
@@ -19,6 +19,16 @@ <h1>{{.Name}}</h1>
           {{ end }}
         </ul>
       </div>
+      {{ if .Form.Action }}
+      <div>
+      <form action="{{ .Form.Action}}">
+      {{ range .Form.Inputs }}
+      <label>{{ .Label }}:</label>&nbsp;<input type="{{ .Type }}" name="{{ .Name }}" placeholder=" .Placeholder }}" value="{{ .Value }}"><br>
```
[the remark this answers is not in the document]
Comment (SuperQ) [reply web/landing_page.html:26]: Oh, oops, will followup.

**Pull request merged**